servers/zms/src/main/java/com/yahoo/athenz/zms/NotificationManager.java

```diff
@@ -175,8 +180,9 @@ public void run() {
         }
 
         private void sendPendingMembershipApprovalReminders() {
+            dbService.updateLastNotifiedTimestamp(pendingRoleMemberLifespan);
```

Comment:
Not sure if I follow how we're protecting multiple hosts processing the same data multiple times. 

host1 - updates all the timestamps.
host2 - tries the same but nothing is updated.
host1 - gets all pending roles
host2 - gets the same pending roles
and then both process notifications.

we might want to consider introducing another column that maintains what host is updating the entry thus that host will get those during get call to process.